windows/gui-source/KA Lite/KALite.cpp

```diff
@@ -282,6 +303,7 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine
 	menu6 = new fle_TrayMenuItem("Run KA Lite at system startup.", &runAtStartupAction);

```

Comment:
Why are we still using ambiguous variable names like `menu6` here instead of `mnuRunAtStartup` or similar?  

I remember having reviewed a PR with these variables which you've already corrected @mrpau-richard - or maybe it's on Kolibri?  If so, then we should propagate those good programming practices back here at KA-Lite.

You can create a separate PR for these changes if you want.